lib/src/model/study/study.dart

```diff
@@ -55,6 +55,16 @@ class Study with _$Study {
     return '${index + 1}. ${chapters[index].name}';
   }
 
+  /// Returns the index of the chapter with the given [chapterId] in the [chapters] list.
+  ///
+  /// The index is 1-based, meaning it starts from 1 instead of 0.
```

Comment:
I'd rather have a 0 based index here, and adds +1 in the view. Will be less confusing.